unifdefall.sh

```diff
@@ -48,7 +48,7 @@ export LC_ALL=C
 # list of all controlling macros; assume these are undefined
 "$unifdef" $debug -s "$@" | sort -u | sed 's/^/#undef /' >"$tmp/undefs"
 # list of all macro definitions
-cc -E -dM "$@" | sort >"$tmp/defs"
+${CC-cc} -E -dM "$@" | sort >"$tmp/defs"
```

Comment:
I suggest ${CC:-cc} -- so that is CC is empty (but set) then cc is used, not ''

Reply:
Sounds good! Updated.